### PR TITLE
street-view-panorama directive

### PR DESCRIPTION
**Pull request by nmccready**
I will add documentation to website-develop once this is merged

### CODE REVIEW

File: example/assets/templates/street-view-window.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div ng-controller="WinController">
+    <ui-gmap-street-view-panorama focalCoord="parameter" radius="50" >
```
Comment (nmccready) [inline example/assets/templates/street-view-window.html]: example just happens to use ui-gmap-window and marker, but it does not require the map at all.

**Comment by nmccready** [comment]
@zacronos

**Comment by zacronos** [comment]
LGTM

**Comment by aogut** [comment]
good job.  thanks.  where can i find the documentation for the street view directive?

**Comment by nmccready** [comment]
@aogut  It just got added to the website today #1572

**Pull request merged**